content/flow-collections/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Control Flow & Collections"
+date: 2023-01-25T13:44:27-06:00
+draft: false
+weight: 1
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
+After completing everything in this chapter, you should be able to do the following:
+1. Become familiar with C# Operators
+   - `==`, `!=`, `<`, `>`, `<=`, and `>=`
+1. Write conditionals using:
+   - `if`, `else`, `if/else` statements
+   - `switch`, `break`, and `continue` statements
+1. Create loops and know when to use them
+   - `for`, `foreach`, `while`, and `do...while`
+1. Know how and when to use list, array, and dictionary collections.
+
+## Key Terminology
```

Comment:
Please update the formatting for this section.

Reply:
done.